Sources/TuistSupport/MachineEnvironment.swift

```diff
@@ -15,18 +15,27 @@ public class MachineEnvironment: MachineEnvironmentRetrieving {
     private init() {}
 
     /// `clientId` is a unique anonymous hash that identifies the machine running Tuist
-    public let clientId = (Host.current().name?.checksum(algorithm: .md5)) ?? "unknown"
+    public lazy var clientId = try! System.shared // swiftlint:disable:this force_try
+        .capture("ioreg", "-d2", "-c", "IOPlatformExpertDevice")
+        .components(separatedBy: "\"IOPlatformUUID\" = \"").last!
+        .components(separatedBy: "\"").first!
+        .checksum(algorithm: .md5)!
```

Comment:
You can do this without an external process. See [here](https://stackoverflow.com/a/55001824).

Reply:
Thanks! Feel free to push the changes, or I'll have a look on Monday 👍

Reply:
Nice! I like that one much more.